client-src/elements/chromedash-feature-table.js

```diff
@@ -338,6 +355,45 @@ class ChromedashFeatureTable extends LitElement {
     return nothing;
   }
 
+  shiftStartBy(delta) {
+    const proposedStart = this.start + delta;
+    this.start = clamp(proposedStart, 0, this.totalCount - 1);
+    this.fetchFeatures();
+  }
+
+  renderPagination() {
```

Comment:
In the case when `this.loading=true` and `return html`<div class="pagination"></div>`;` (line382) is rendered, what data binding will trigger `renderPagination` to reload and show line 382?

Reply:
`this.loading` is a property of `ChromedashFeatureTable`.  'this.loading' is set true by `fetchFeatures()` when a request to the server is initiated and then set to false when the response has been processed.  Any change to a Lit element property causes `render()` to eventually be called, which in turn calls `renderPagination()`.